src/main/java/net/mrscauthd/beyond_earth/events/Methods.java

```diff
@@ -357,7 +335,7 @@ public static void landerTeleport(Player player, ResourceKey<Level> newPlanet) {
         }
     }
 
-    public static void rocketTeleport(Player player, ResourceKey<Level> planet, ItemStack rocketItem, Boolean SpaceStation) {
+    public static void rocketTeleport(Player player, ResourceKey<Level> planet, ItemStack rocketItem, boolean SpaceStation) {
```

Comment:
Use `boolean` (lowercase) instead, it's faster than `Boolean` (uppercase)`.

`Boolean` (uppercase) is slow and usually unnecessary.